composer.json

```diff
@@ -109,6 +109,7 @@
     "scripts": {
         "phing-install-dependencies": ["phing patch-dependencies", "phing installsolr installswaggerui"],
         "post-install-cmd": "@phing-install-dependencies",
-        "post-update-cmd": "@phing-install-dependencies"
+        "post-update-cmd": "@phing-install-dependencies",
+        "qa": "phing eslint phpunitfast phpcs-console php-cs-fixer-dryrun"
```

Comment:
For consistency, I would make this `phing qa-console` so that if we add more tasks, we only need to put them in one place.

Reply:
Yeah, forgot that one and just changed it!